lemur-build-docker/files/entrypoint

```diff
@@ -48,10 +39,13 @@ elif [ "${POSTGRES_DB_MODE}" = "init" ]; then
 	PGPASSWORD=$POSTGRES_PASSWORD dropdb -h $POSTGRES_HOST -p $POSTGRES_PORT -U $POSTGRES_USER $POSTGRES_DB --if-exists
 	PGPASSWORD=$POSTGRES_PASSWORD createdb -h $POSTGRES_HOST -p $POSTGRES_PORT -U $POSTGRES_USER $POSTGRES_DB -T template0
 	cd /opt/lemur/lemur
-	python3 /opt/lemur/lemur/manage.py --config=/home/lemur/.lemur/lemur.conf.py init -p "${LEMUR_ADMIN_PASSWORD}"
+	/opt/venv/bin/python3 /opt/lemur/lemur/manage.py --config=/home/lemur/.lemur/lemur.conf.py init -p "${LEMUR_ADMIN_PASSWORD}"
 	echo " # Lemur has been initialized with admin username 'lemur' and the specified password"
 else
 	echo " # POSTGRES_DB_MODE was neither 'load-from-dump' nor 'init', so will use existing database"
+	# for first time use, init should not touched exists data
+	cd /opt/lemur/lemur
+	/opt/venv/bin/python3 /opt/lemur/lemur/manage.py --config=/home/lemur/.lemur/lemur.conf.py init -p "${LEMUR_ADMIN_PASSWORD}"
```

Comment:
Did you find a case that makes this necessary? My intent with the default/unset case was to leave the DB completely alone (and that if you need initialization, you would set `POSTGRES_DB_MODE=init`).

Reply:
What I need to do if I need persistent db?

Before changes:
- set POSTGRES_DB_MODE=init
- start containers
- stop containers
- remove POSTGRES_DB_MODE=init
- start containers

After changes:
- start containers

Reply:
I general case I think entrypoint should  be very simple and clear. If you need some logic it can be made in Makefile:
```
lemur_checkout:
	[ -d $(LEMUR_GIT_DIR) ] || git clone --depth=1 https://github.com/Netflix/lemur.git $(LEMUR_GIT_DIR)
init_db:
       // some logic
load_from_dump_db:
    // some logic
```

Reply:
That's reasonable, I agree.